common/src/main/java/io/druid/common/utils/UUIDUtils.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2012 - 2015 Metamarkets Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.common.utils;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+
+import java.util.ArrayList;
+import java.util.UUID;
+
+/**
+ *
+ */
+public class UUIDUtils
+{
+  public static final String UUID_DELIM = "-";
```

Comment:
right, but so far everything in Druid typically uses _ for separators until now, for instance segment IDs, taks IDs. We can just remove the '-' from the UUIDs, since they are not really necessary.